spark-plugins/src/main/java/co/cask/hydrator/plugin/spark/Tokenizer.java

```diff
@@ -45,13 +50,47 @@
 public class Tokenizer extends SparkCompute<StructuredRecord, StructuredRecord> {
 
   public static final String PLUGIN_NAME = "Tokenizer";
+  private static Schema outputSchema;
+  private static Schema inputSchema;
+  private static List<String> fieldList = new ArrayList<>();
+
   private Config config;
-  private Schema outputSchema;
 
   public Tokenizer(Config config) {
     this.config = config;
   }
 
+  private static DataType getDataType(Schema schema) {
```

Comment:
It would be nice to have the CDAP platform support of taking `RDD<StructuredRow>` and turning it into `DataFrame`. It's pretty standard and can be used by every Spark programs and plugins. Can you open a JIRA to track that?